guides/common/assembly_importing-kickstart-repositories.adoc

```diff
@@ -4,7 +4,6 @@ include::modules/con_importing-kickstart-repositories.adoc[]
 
 :rhel-version: 7
 :context: rhel-7
-include::modules/proc_importing-kickstart-repositories.adoc[leveloffset=+1]
```

Comment:
AFAIK this is only for clients, which remains supported.

Reply:
Addressed.